tests/power/devicepm/common/test_powerstate.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
```

Comment:
Should copyright year be updated to latest one?

Reply:
Done